conans/test/functional/toolchains/meson/_base.py

```diff
@@ -30,7 +31,7 @@ def _settings(self):
                             "build_type": "Release"}
 
         settings_linux = {"compiler": "gcc",
-                          "compiler.version": "5",
+                          "compiler.version": os.getenv('CONAN_COMPILER_VERSION'),
```

Comment:
We are not using CONAN_COMPILER_VERSION for the system compiler anymore in CI, this might fail in CI. Conan 2.0 has already changed this using a defined ``default_profiles`` variable with full profiles, but at the moment this might need to keep hardcoded for CI maybe.

Reply:
I see, it does not work. Hope it will be better soon.